pkg/services/ngalert/schedule/schedule_unit_test.go

```diff
@@ -504,7 +504,7 @@ func TestSchedule_ruleRoutine(t *testing.T) {
 		// Now update the rule
 		newRule := *rule
 		newRule.Version++
-		ruleStore.putRule(&newRule)
+		ruleStore.putRule(context.TODO(), &newRule)
```

Comment:
Why do we use in the other tests `.Background()` and here `.TODO()`? Is there something special we need to think of in the future?

Reply:
Thanks! I was meant to change these to `Background()`!